tracer/build/_build/Build.Profiler.Steps.cs

```diff
@@ -56,6 +64,7 @@ partial class Build
             // Build native profiler assets
             MSBuild(s => s
                 .SetTargetPath(project)
+                .SetRestore(true)
```

Comment:
I don't think this is necessary if you are doing the explicit restore before. Also, this conflicts with the `DisableRestore()` below!

Reply:
Oh I forgot to remove that.
Thanks for pointing this out.